src/hovedside/maalLinje/mitt-maal.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import PT from 'prop-types';
+import { connect } from 'react-redux';
+import Tekstomrade from 'nav-frontend-tekstomrade';
+import { FormattedMessage } from 'react-intl';
+import { Element } from 'nav-frontend-typografi';
+import Lenke from '../../felles-komponenter/utils/lenke';
+import mittMalSvg from './Illustrasjon_dette_gjor_du_bra.svg';
+import Innholdslaster from '../../felles-komponenter/utils/innholdslaster';
+import { selectErVeileder } from '../../moduler/identitet/identitet-selector';
+import { div as HiddenIfDiv } from '../../felles-komponenter/hidden-if/hidden-if';
+import {
+    hentMal,
+    selectGjeldendeMal,
+    selectMalStatus,
+} from '../../moduler/mal/aktivitetsmal-reducer';
+import * as AppPT from '../../proptypes';
+
+const Mal = ({ mal }) => {
```

Comment:
Ellers i løsningen brukes syntaksen `function Mal({ mal }) {  }` for å lage SFC, så da burde vi nok gjøre det her også imho